.concourse/config.yaml.sample

```diff
@@ -0,0 +1,11 @@
+# All null (~) values have defaults in the pipeline
```

Comment:
How can we absolutely make sure that there is no/minimal possibility for secret leaks? As the pipeline might be having public access.

Reply:
We removed the secrets from the values here. You are right, we don't need to templetize them. If someone is deploying on a different concourse instance, they will simpy add their own secrets in their secret storage solution.